[Bug]: Failed to build package 'apt' for arch 'aarch64'

### Problem description

[build.log](https://github.com/termux/termux-packages/files/13718382/build.log)
```
ld.lld: error: version script assignment of 'LIBCTF_1.0' to symbol 'ctf_label_set' failed: symbol not defined
ld.lld: error: version script assignment of 'LIBCTF_1.0' to symbol 'ctf_label_get' failed: symbol not defined
clang-17: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [Makefile:687: libctf.la] Error 1
make[3]: *** Waiting for unfinished jobs....
(for i in `seq 1 1000 70000`; do \
   echo "assert(var_$i == $i);"; \
 done) > many_sections_check.h.tmp
mv -f many_sections_check.h.tmp many_sections_check.h
  CC       call_graph.o
  CC       cg_arcs.o
  CC       cg_dfn.o
  CC       cg_print.o
  CC       corefile.o
make[3]: Leaving directory '/home/builder/.termux-build/binutils-libs/build/libctf'
make[2]: *** [Makefile:583: all] Error 2
make[2]: Leaving directory '/home/builder/.termux-build/binutils-libs/build/libctf'
make[1]: *** [Makefile:12000: all-libctf] Error 2
make[1]: *** Waiting for unfinished jobs....
```
note: I changed the TERMUX_APP_PACKAGE in [scripts/properties.sh](https://github.com/termux/termux-packages/blob/master/scripts/properties.sh) to a custom package name.

### What steps will reproduce the bug?

`./clean.sh
./scripts/run-docker.ps1 ./scripts/build-bootstraps.sh`


### What is the expected behavior?

successfully  build bootstrap archives for the termux-app.

### System information

termux-info:

N/A

--- a/packages/binutils-libs/build.sh
+++ b/packages/binutils-libs/build.sh
@@ -49,6 +49,14 @@ termux_step_pre_configure() {
 	rm -rf $TERMUX_HOSTBUILD_MARKER
 
 	export CPPFLAGS="$CPPFLAGS -Wno-c++11-narrowing"
+	# llvm upgraded a warning to an error, which caused this build (and some
+	# others, including the rust toolchain) to fail like so:
+	#
+	# ld.lld: error: version script assignment of 'LIBCTF_1.0' to symbol 'ctf_label_set' failed: symbol not defined
+  # ld.lld: error: version script assignment of 'LIBCTF_1.0' to symbol 'ctf_label_get' failed: symbol not defined
+	# These flags restore it to a warning.
+	# https://reviews.llvm.org/D135402
+	export LDFLAGS="$LDFLAGS -Wl,--undefined-version"
 
 	if [ $TERMUX_ARCH_BITS = 32 ]; then
 		export LIB_PATH="${TERMUX_PREFIX}/lib:/system/lib"

